drop-down.ios.ts

```diff
@@ -362,8 +370,11 @@ class DropDownListPickerDelegateImpl extends NSObject implements UIPickerViewDel
         };
 
         label.font = style.fontInternal.getUIFont(label.font);
-
-        switch (style.textAlignment) {
+        let p_itemsTextAlignment = owner.itemsTextAlignment;
```

Comment:
change naming to be consistent with the rest of the code. So this should be just `itemsTextAlignment`